CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Resolver
 - Fixed: Bug where nested requirements were combined wrongly.
+- Fixed: Bug where damage constraints in chains were not understood correctly.
```

Comment:
You should also add a changelog entry that prime doesnt have multiplicative suit reductions anymore.